testing/test_config.py

```diff
@@ -634,41 +634,37 @@ def pytest_addoption(parser):
         assert val == "hello"
         pytest.raises(ValueError, config.getini, "other")
 
-    def make_conftest_for_pathlist(self, pytester: Pytester) -> None:
+    @pytest.mark.parametrize("config_type", ["ini", "pyproject"])
+    @pytest.mark.parametrize("ini_type", ["paths", "pathlist"])
+    def test_addini_paths(
+        self, pytester: Pytester, config_type: str, ini_type: str
+    ) -> None:
         pytester.makeconftest(
-            """
+            f"""
             def pytest_addoption(parser):
-                parser.addini("paths", "my new ini value", type="pathlist")
+                parser.addini("paths", "my new ini value", type="{ini_type}")
                 parser.addini("abc", "abc value")
         """
         )
-
-    def test_addini_pathlist_ini_files(self, pytester: Pytester) -> None:
-        self.make_conftest_for_pathlist(pytester)
-        p = pytester.makeini(
+        if config_type == "ini":
+            inipath = pytester.makeini(
```

Comment:
lets use the name config_path for clarity

Reply:
The "official" name is `inipath` (as in `config.inipath`) so I used that.